homeassistant/components/bluetooth/scanner.py

```diff
@@ -140,6 +140,7 @@ def __init__(
         self._new_info_callback = new_info_callback
         self.scanning = False
         self.hass = hass
+        self._last_detection = 0.0
```

Comment:
I suppose this one of the moved parameters?

Reply:
Hmm, it seems to be initialized in the base class. The linters do not pick that up so it seems.

Reply:
I'm not 100% sure whats going on there, but it does work as expected, and I think it won't matter so much since once I move it, it doesn't seem to error anymore.